knimin/lib/data_access.py

```diff
@@ -962,7 +970,10 @@ def _decode_zip_lookup(item):
                         'SURVEY_ID'], unknown_external))
             except Exception as e:
                 # Add barcode to error and remove from metadata info
-                errors[barcode] = '%s' % e
+                if isinstance(e, SSLError):
```

Comment:
I am surprised about this - SSLError should be a subclass of exception, so it should have the `message` attribute. Why the special handling?